examples/ads1115/main/main.c

```diff
@@ -3,17 +3,26 @@
 #include <freertos/task.h>
 #include <ads111x.h>
 #include <string.h>
-
-#define DEV_COUNT 2   // 2 ICs
+// Make the connections as below:
+/*
+| S. No. | ADS 1115 | ESP 32            |
+|--------|----------|-------------------|
+| 1.     | V_DD     | V_in or 5V source |
+| 2.     | GND      | GND               |
+| 3.     | SCL      | D22               |
+| 4.     | SDA      | D21               |
+| 5.     | A0       | analog input      |
+*/
+#define DEV_COUNT 1   // 1 IC
 
 #define I2C_PORT 0
 
 #if defined(CONFIG_IDF_TARGET_ESP8266)
 #define SDA_GPIO 4
 #define SCL_GPIO 5
 #else
-#define SDA_GPIO 16
-#define SCL_GPIO 17
+#define SDA_GPIO 21     
+#define SCL_GPIO 22
 #endif
```

Comment:
I see no reason to change the GPIO pins.

Reply:
alright
But atleast provide the pin diagram which is used in your example, it will be of great help thanks!